lib/haml_lint/severity.rb

```diff
@@ -0,0 +1,69 @@
+require 'delegate'
+
+module HamlLint
+  # Models the severity of a lint
+  class Severity < SimpleDelegator
+    include Comparable
```

Comment:
I like how you're using Comparable here so that you can use `<=` to compare different ones. Neat.